Cargo.toml

```diff
@@ -201,6 +201,7 @@ slow_tests = []
 std = ["alloc"]
 test_logging = []
 wasm32_unknown_unknown_js = ["web-sys"]
+size_optimized = []
```

Comment:
I'd rather not control this with a feature flag. Perhaps you can work around this with `CFLAGS` in your build environment for now?